gradle.properties

```diff
@@ -17,6 +17,9 @@ org.gradle.jvmargs=-Xmx4096m
 # XXX Deactivated because of broken releases?
 org.gradle.parallel=false
 
+# Build cache - further configured in settings.gradle
+org.gradle.caching=true
```

Comment:
The only fear I have here is that we may have non-standard tasks that are not cacheable. Or that we cache tasks that actually should run everytime. Like things that depend on timestamps or constants in BuildConfig. Can you do some sanity checks to make sure they look like what you'd expect? 🤔 
Does this have any impact on our custom substitution build setup?

Reply:
This shouldn't affect autopublication stuff, but I'll verify that.

Build scans list a few hundred of our tasks as non-cacheable already. I'll do a sanity pass to see if something that's marked as cacheable stands out to me as "shouldn't be".

Otherwise, we should benefit from going through the non-cacheable ones and figuring out if they can be/should be cached. But that's a follow-up.